omero/sysadmins/server-upgrade.rst

```diff
@@ -272,6 +272,26 @@ run only on the specific version before the next upgrade script.
    using the above command with
    :file:`sql/psql/OMERO5.4__0/OMERO5.3__0.sql`.
 
+Remove the guest user password (optional)
+"""""""""""""""""""""""""""""""""""""""""
+
+If a password was set on the `guest` user to work around
+:secvuln:`2017-SV4-guest-user` then you may now wish to remove it so
+that a correct password is not needed to log in as that user:
+
+.. parsed-literal::
+
+    $ psql -h localhost -U **db_user** **omero_database** < sql/psql/|current_dbver|/allow-guest-user-without-password.sql
+
+This can be done at any time during the OMERO 5.4 series.
+
+.. note::
+
+    The above script assumes that the `guest` user has an ID of 1. In
+    the unusual case of that not being so then the script will do
```

Comment:
This is a bit of a mouthful, could we just say "If that is not true then the script..."

Reply:
I *think* it's so unlikely that this note will help approximately no readers at all. I'll figure a way to simplify.